src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs

```diff
@@ -155,10 +151,34 @@ public void InitializeService(ServiceHost serviceHost)
             }
         }
 
-        /// <summary>
-        /// Handles request to start the scripting operation
-        /// </summary>
-        public async Task HandleScriptExecuteRequest(ScriptingParams parameters, RequestContext<ScriptingResult> requestContext)
+        private void RunScriptAsTask(ConnectionInfo connInfo, ScriptingParams parameters, RequestContext<ScriptingResult> requestContext)
+        {
+            ScriptAsScriptingOperation operation = new ScriptAsScriptingOperation(parameters);
+            ConnectionServiceInstance.ConnectionQueue.QueueBindingOperation(
+                key: ConnectionServiceInstance.ConnectionQueue.AddConnectionContext(connInfo, "Scripting"),
+                bindingTimeout: ScriptingOperationTimeout,
+                bindOperation: (bindingContext, cancelToken) =>
+                {
+                    string script = string.Empty;
```

Comment:
this seems unused? Can you remove?

Reply:
I'm changing this code in another PR 

---
In reply to: [154441355](https://github.com/Microsoft/sqltoolsservice/pull/556#discussion_r154441355) [](ancestors = 154441355)